localstack/testing/pytest/fixtures.py

```diff
@@ -1131,6 +1131,154 @@ def _check_logs(func_name: str, expected_lines: List[str] = None):
     return _check_logs
 
 
+@pytest.fixture
+def wait_for_user():
```

Comment:
probably better suited to be put into the aws testing utils since it doesn't really have any setup/teardown context